L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigis_cfi.py

```diff
@@ -510,6 +510,18 @@
                            )
 
 ## GEM-CSC ILT in ME1/1
+from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
+run2_GEM_2017.toModify( cscTriggerPrimitiveDigis,
+                        GEMPadDigiProducer = cms.InputTag("simMuonGEMPadDigis"),
```

Comment:
@slava77 I think you can eschew the new Era if it's just a modifier that will be used for private tests. The new modifier can be appended to the usual `Run2_2017` Era (as a comma-separated list) on the fly in a cmsDriver command or cmsRun config.